templates/components/base/global_nav.html.twig

```diff
@@ -1,767 +1,62 @@
+						<span class="logo">
+							<img src="{{ asset('svg/w3c-logo.svg', 'main') }}" alt="W3C logo" role="img"/>
+						</span>
             <span class="visuallyhidden">W3C homepage</span>
         </a>
-        <button type="button" class="button button--ghost with-icon--after with-icon--larger"
-                data-trigger="mobile-nav"></button>
+        <button type="button" class="button button--ghost with-icon--after with-icon--larger" data-trigger="mobile-nav"
+                style="display: none;"></button>
         <ul class="clean-list">
-            <li class="top-nav-item has-children">
-                <a href="../page.html" class="nav-link">Standards</a>
+            {% for navItem in navigation %}
+                <li class="top-nav-item has-children">
+                    <a href="{{ navItem.titleLink }}" class="nav-link">{{ navItem.title }}</a>
                 <div class="nav__submenu" style="display: none;">
                     <div class="l-center">
                         <div class="nav__submenu__intro">
                             <div class="nav__submenu__intro__text">
-                                <p>Understand the various specifications, their maturity levels on the Web
-                                    Standards track, and their adoption.</p>
-                                <a class="with-icon--before with-icon--larger" href="../page.html"
-                                   aria-hidden="true" tabindex="-1">
-                                    <svg class="icon icon--larger" xmlns="http://www.w3.org/2000/svg"
-                                         focusable="false" aria-hidden="true" viewBox="0 0 320 512"
-                                         width="1em" height="1em">
-                                        <path class="chevron-right"
-                                              d="M285.476 272.971L91.132 467.314c-9.373 9.373-24.569 9.373-33.941 0l-22.667-22.667c-9.357-9.357-9.375-24.522-.04-33.901L188.505 256 34.484 101.255c-9.335-9.379-9.317-24.544.04-33.901l22.667-22.667c9.373-9.373 24.569-9.373 33.941 0L285.475 239.03c9.373 9.372 9.373 24.568.001 33.941z"/>
-                                        <path class="chevron-left"
-                                              d="M34.52 239.03L228.87 44.69c9.37-9.37 24.57-9.37 33.94 0l22.67 22.67c9.36 9.36 9.37 24.52.04 33.9L131.49 256l154.02 154.75c9.34 9.38 9.32 24.54-.04 33.9l-22.67 22.67c-9.37 9.37-24.57 9.37-33.94 0L34.52 272.97c-9.37-9.37-9.37-24.57 0-33.94z"/>
-                                    </svg>
-                                    Explore Web Standards</a>
+                                <p>{{ navItem.introText }}</p>
```

Comment:
Is it ever possible that `introText` might be empty? It is not currently a required field in the CMS (if it was empty at the moment, it would output an empty `<p>` element).

Reply:
I've added a safety net